### ISSUE TITLE
docs: example of incorrect code for `no-multiple-empty-lines` rule is actually correct

### ISSUE THREAD

**Issue by jonz94** [eslint/eslint#16832]
### ESLint version

v8.32.0

### What problem do you want to solve?

Currently, the first example of incorrect code for `no-multiple-empty-lines` rule with default option is **actually correct**.

#### max

Docs URL: https://eslint.org/docs/latest/rules/no-multiple-empty-lines#max

Examples of **incorrect** code for this rule with the default `{ "max": 2 }` option: 

```javascript
/*eslint no-multiple-empty-lines: "error"*/

var foo = 5;


var bar = 3;
```

[ESLint Playground of above example code](https://eslint.org/play/#eyJ0ZXh0IjoiLyplc2xpbnQgbm8tbXVsdGlwbGUtZW1wdHktbGluZXM6IFwiZXJyb3JcIiovXG5cbnZhciBmb28gPSA1O1xuXG5cbnZhciBiYXIgPSAzOyIsIm9wdGlvbnMiOnsicGFyc2VyT3B0aW9ucyI6eyJlY21hVmVyc2lvbiI6ImxhdGVzdCIsInNvdXJjZVR5cGUiOiJzY3JpcHQiLCJlY21hRmVhdHVyZXMiOnt9fSwicnVsZXMiOnt9LCJlbnYiOnsiZXM2Ijp0cnVlfX19) shows this code is correct.

### What do you think is the correct solution?

I'd like to change the examples as the following:

#### max

Examples of **incorrect** code for this rule with the default `{ "max": 2 }` option:

```javascript
/*eslint no-multiple-empty-lines: "error"*/

var foo = 5;



var bar = 3;
```

Examples of **correct** code for this rule with the default `{ "max": 2 }` option:

```javascript
/*eslint no-multiple-empty-lines: "error"*/

var foo = 5;


var bar = 3;
```

### Participation

- [X] I am willing to submit a pull request for this change.

**Comment by mdjermanovic** [comment]
Hi @jonz94, thanks for the issue!

It seems that empty lines that are important for these examples were accidentally removed in https://github.com/eslint/eslint/commit/db28f2c9ea6b654f615daf2f7e6f1a2034b85062. PR to fix this is welcome!
